sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_registry_artifact.py

```diff
@@ -357,7 +181,7 @@ def list_tags(self, **kwargs: Dict[str, Any]) -> AsyncItemPaged[ArtifactTagPrope
             call will return values after last lexically
         :paramtype last: str
         :keyword order_by: Query parameter for ordering by time ascending or descending
-        :paramtype order_by: :class:`~azure.containerregistry.TagOrderBy`
+        :paramtype order_by: :class:`~azure.containerregistry.TagOrder`
```

Comment:
The param documentation should include `str` as a valid input where ever we also use enums :)
Though it doesn't need to be fixed for this preview.